python/lsst/ts/xml/data/sal_interfaces/MTDome/MTDome_Events.xml

```diff
@@ -287,6 +293,7 @@
   <SALEvent>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_logevent_brakesEngaged</EFDB_Topic>
+    <Description>Brakes engaged.</Description>
```

Comment:
Consider making the description a bit more descriptive. We can certainly get the "brakes engaged" part from the topic name. Consider something like:

```
Reports whether the brakes are engaged or not.
```